docs/UserGuide.md

```diff
@@ -175,8 +175,7 @@ Example:
 
 * `clear` clears all Client's entries together with the Remarks and Transactions.
 
-> <div markdown="span" class="alert alert-danger">❗ :warning: Information cleared by clear command cannot be retrie
- </div>
+> <div markdown="span" class="alert alert-danger">❗ :warning: Information cleared by clear command cannot be retrieved
```

Comment:
Good find! Would have been a avoidable bug :)

Reply:
I agree

Reply:
I added an exclamation mark in the latest commit!

Reply:
nice! makes the alert look better